hack/perf-ibmcloud-openshift.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+##############################################################################
+# perf-ibmcloud-openshift.sh
+#
+# Setup a performance testing environment on Openshift in IBM Cloud.
+#
+# Pass --help for usage information.
+#
+##############################################################################
+
+set -eu
+
+errormsg() {
+  echo -e "\U0001F6A8 ERROR: ${1}"
+  exit 1
+}
+
+infomsg() {
+  echo -e "\U0001F4C4 ${1}"
+}
+
+install_istio() {
+	hack/istio/download-istio.sh -iv "1.13.0"
```

Comment:
TAB!
```suggestion
  hack/istio/download-istio.sh -iv "1.13.0"
```

Reply:
finally, do we want this fixed on istio 1.13? 1.14 is out.

Perhaps don't provide any version - the download script will download the latest release, so no need to hardcode a value here.
```suggestion
  hack/istio/download-istio.sh
```

Reply:
> There are also lines with spaces before EOL. Ewwww...

I've become reliant on my IDE to spot/fix these things for me :)

> finally, do we want this fixed on istio 1.13? 1.14 is out.

Yeah good catch. That was just leftover from me testing things.